cmd/skupper/skupper.go

```diff
@@ -440,10 +441,22 @@ installation that can then be connected to other skupper installations`,
 				return fmt.Errorf("The --enable-flow-collector option must be used with the --enable-console option")
 			}
 
+			if routerCreateOpts.AuthMode != "internal" && (len(routerCreateOpts.User) > 0 || len(routerCreateOpts.Password) > 0) {
+				return fmt.Errorf("for the console to work with this user or password, the --console-auth option must be set to internal")
+			}
+
+			if len(routerCreateOpts.AuthMode) > 0 && !routerCreateOpts.EnableFlowCollector {
+				fmt.Println("The authentication only applies when --enable-console or --enable-flow-collector are enabled")
```

Comment:
Is it intentional just to show this message and let it move on?

Reply:
It is intentional,  it was agreed in [this conversation.](https://github.com/skupperproject/skupper/pull/1152#discussion_r1251837114)